solana/solana-ibc/programs/solana-ibc/src/tests.rs

```diff
@@ -795,6 +795,77 @@ fn anchor_test_deliver() -> Result<()> {
         })?;
     println!("  Signature {sig}");
 
+    /*
+     * Realloc Accounts
+     */
+
+    let storage_acc_length_before =
+        sol_rpc_client.get_account(&storage).unwrap();
+
+    println!("\nReallocating Accounts");
+    let sig = program
+        .request()
+        .accounts(accounts::ReallocAccounts {
+            payer: authority.pubkey(),
+            account: storage,
+            rent: anchor_lang::solana_program::sysvar::rent::ID,
+            system_program: system_program::ID,
+        })
+        .args(instruction::ReallocAccounts {
+            // we can increase upto 10kb in each tx so increasing it to 20kb since 10kb was already allocated
+            new_length: 2 * (1024 * 10),
+        })
+        .payer(authority.clone())
+        .signer(&*authority)
+        .send_with_spinner_and_config(RpcSendTransactionConfig {
+            skip_preflight: true,
+            ..RpcSendTransactionConfig::default()
+        })?;
+    println!("  Signature {sig}");
+
+    let storage_acc_length_after =
+        sol_rpc_client.get_account(&storage).unwrap();
+
+    assert_eq!(
+        storage_acc_length_after.data.len() -
+            storage_acc_length_before.data.len(),
+        1024 * 10
+    );
```

Comment:
IMO it’s more obvious to just check the new length and not the difference.

```suggestion
    assert_eq!(20 * 1024, storage_acc_length_after.data.len());
```

Reply:
true